business_requirement_deliverable_cost/models/business.py

```diff
@@ -200,21 +223,30 @@ def _compute_rl_total_cost(self):
     def _compute_resource_task_total(self):
         for br in self:
             if br.deliverable_lines:
-                br.resource_task_total =\
+                resource_task_total =\
                     sum(br.mapped('deliverable_lines').mapped('resource_ids')
                         .filtered(lambda r: r.resource_type == 'task')
                         .mapped('price_total'))
+                resource_task_total_ci = self.env['res.currency']._compute(
+                    br.currency_id, self.env.user.company_id.currency_id,
+                    resource_task_total)
+                br.resource_task_total = resource_task_total_ci
```

Comment:
@YogeshMahera-SerpentCS Try not use intermediate variable `resource_task_total_ci` just apply the new calculation.

Multiple places I've see so please have a check